icons/verified.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M11.997 3a3.599 3.599 0 00-3.044 1.682 3.599 3.599 0 00-.908-.095 3.599 3.599 0 00-2.411 1.051 3.6 3.6 0 00-.947 3.318A3.6 3.6 0 003 12.004a3.6 3.6 0 001.69 3.047 3.6 3.6 0 00.947 3.315 3.599 3.599 0 003.352.96A3.599 3.599 0 0012.002 21a3.599 3.599 0 003.037-1.671 3.599 3.599 0 004.295-4.297A3.6 3.6 0 0021 11.998a3.6 3.6 0 00-1.67-3.035v-.002a3.599 3.599 0 00-4.298-4.292A3.599 3.599 0 0011.997 3z" />
```

Comment:
With [SVGOMG](https://jakearchibald.github.io/svgomg/) I was able to minify it a bit.

```suggestion
  <path d="M12 3a3.6 3.6 0 0 0-3.05 1.68 3.6 3.6 0 0 0-.9-.1 3.6 3.6 0 0 0-2.42 1.06 3.6 3.6 0 0 0-.94 3.32A3.6 3.6 0 0 0 3 12a3.6 3.6 0 0 0 1.69 3.05 3.6 3.6 0 0 0 .95 3.32 3.6 3.6 0 0 0 3.35.96A3.6 3.6 0 0 0 12 21a3.6 3.6 0 0 0 3.04-1.67 3.6 3.6 0 0 0 4.3-4.3A3.6 3.6 0 0 0 21 12a3.6 3.6 0 0 0-1.67-3.04v0a3.6 3.6 0 0 0-4.3-4.3A3.6 3.6 0 0 0 12 3z"/>
```

Reply:
Ah yeah. I forgot about that.
Anyway, can you guide me how to use SVGO? I wanted to try it myself, but the result was different from what you suggested.